### PR TITLE
Update status icon according to routeType status

### PR DESCRIPTION
**Pull request by pblottiere**

### CODE REVIEW

**Comment by pblottiere** [comment]
@SebastienPeillet can you take a quick look please?

File: src/gui/status.py [hunk 1]
```diff
@@ -145,6 +148,33 @@ def updateGpsInfo(self, longitude: float, latitude: float):
...
+        idx = layer.fields().indexFromName("routeType")
+        for feature in layer.getFeatures():
+            routeType = feature[idx]
+            if routeType != "prospection":
+                continue
+
+            if not feat:
+                feat = feature
+            elif feature.id() > feat.id():
+                feat = feature
```
Comment (SebastienPeillet) [inline src/gui/status.py:167]:
```suggestion
        request = QgsFeatureRequest("routeType = 'prospection'")
        request.addOrderBy("fid", false)
        for feat in layer.getFeatures(request):
            break
```

Comment (pblottiere) [reply src/gui/status.py:167]: Yep, much better :)

**Comment by SebastienPeillet** [comment]
@pblottiere : I added a suggestion for using QgsFeatureRequest instead of looping on each feature. Otherwise it's good to me !

File: src/gui/status.py [hunk 2]
```diff
@@ -145,6 +148,28 @@ def updateGpsInfo(self, longitude: float, latitude: float):
...
         self._endThread()
 
+    @property
+    def _isEffortOn(self) -> bool:
+        layer = self.session.environmentLayer
+        if not layer:
+            return False
+
+        feat = None
+        idx = layer.fields().indexFromName("routeType")
```
Comment (SebastienPeillet) [inline src/gui/status.py:158]:
```suggestion
```

**Comment by pblottiere** [comment]
Thanks for the review :+1:

**Pull request merged**